wrangler-core/src/main/java/io/cdap/functions/NumberFunctions.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright © 2020 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+package io.cdap.functions;
+
+public final class NumberFunctions {
+
+  /**
+   * @return The number as double.
+   */
+  public static double AsDouble(Number value) {
```

Comment:
if a null is passed in, this will error. What is the expected behavior for nulls?

Reply:
Added handling of null, the method will now return null